docs/source/installation.rst

```diff
@@ -88,7 +88,7 @@ Inside the development container perform the following steps.
       .. code-block:: bash
 
           cd /app/data/US/
-          /usr/src/tensorrt/bin/trtexec --onnx=us_unet_256x256.onnx --saveEngine=model.engine --explicitBatch --verbose --workspace=1000
+          /usr/src/tensorrt/bin/trtexec --onnx=us_unet_256x256.onnx --saveEngine=model.engine --explicitBatch --verbose --workspace=5000
```

Comment:
5000 will fail in most development devices that's why I set 1000 to give it a swimming chance

Reply:
@aihsani 5000 is specific to the AGX.

Reply:
1000 worked on AGX, reverting.